From: Andreas Gruenbacher <agruen@suse.de>
Subject: Remove EXTRAVERSION

The KERNELRELEASE has become too long for the installation-images package;
we must shorten it quickly for now.

Signed-off-by: Andreas Gruenbacher <agruen@suse.de>

Index: linux-2.6.10/Makefile
===================================================================
--- linux-2.6.10.orig/Makefile
+++ linux-2.6.10/Makefile
@@ -1,7 +1,7 @@
 VERSION = 2
 PATCHLEVEL = 6
 SUBLEVEL = 11
-EXTRAVERSION = -rc2-bk10
+EXTRAVERSION =
 NAME=Woozy Numbat
 
 # *DOCUMENTATION*
